server/sonar-web/src/main/js/apps/projects/store/actions.js

```diff
@@ -68,37 +68,34 @@ const FACETS = [
   'tags'
 ];
 
-const onFail = dispatch =>
-  error => {
-    parseError(error).then(message => dispatch(addGlobalErrorMessage(message)));
-    dispatch(updateState({ loading: false }));
-  };
+const onFail = dispatch => error => {
+  parseError(error).then(message => dispatch(addGlobalErrorMessage(message)));
+  dispatch(updateState({ loading: false }));
+};
 
-const onReceiveMeasures = (dispatch, expectedProjectKeys) =>
-  response => {
-    const byComponentKey = groupBy(response.measures, 'component');
+const onReceiveMeasures = (dispatch, expectedProjectKeys) => response => {
+  const byComponentKey = groupBy(response.measures, 'component');
 
-    const toStore = {};
+  const toStore = {};
 
-    // fill store with empty objects for expected projects
-    // this is required to not have "null"s for provisioned projects
-    expectedProjectKeys.forEach(projectKey => toStore[projectKey] = {});
+  // fill store with empty objects for expected projects
+  // this is required to not have "null"s for provisioned projects
+  expectedProjectKeys.forEach(projectKey => (toStore[projectKey] = {}));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract the assignment of "toStore[projectKey]" from this expression. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)